wasm.md

```diff
@@ -193,13 +193,14 @@ This allows us to give purely functional semantics to many of the opcodes.
 
 Constants are moved directly to the value stack.
 Function `#unsigned` is called on integers to allow programs to use negative numbers directly.
+**TODO**: Implemented Float in the format of `-nan`, `nan:0x n:hexnum` and `hexfloat`.
 
 ```k
     syntax PlainInstr ::= IValType "." "const" Int
-                        | FValType "." "const" Float
- // ------------------------------------------------
-    rule <k> ITYPE:IValType . const VAL => #chop(< ITYPE > VAL) ... </k>
-    rule <k> FTYPE:FValType . const VAL => < FTYPE > VAL        ... </k>
+                        | FValType "." "const" Number
```

Comment:
Is this because a float can given as a number with no `.` in wasm, but it has to have a `.` in K?

Reply:
Yes. In K if a token is parsed as Int, then it cannot be parsed as Float. So the ambiguity is not allowed and it is pretty annoying.